### PR TITLE
filesystem for clix 

### PR DESCRIPTION
**Pull request by akansh97531**
@nikhilkumarsingh please look
and I had to move variable clips to utils , beacuse i needed clips in gui.py

### CODE REVIEW

**Comment by akansh97531** [comment]
@nikhilkumarsingh

**Comment by akansh97531** [comment]
@nikhilkumarsingh

File: clix/gui.py [hunk 1]
```diff
@@ -1,103 +1,122 @@
...
+    from ScrolledText import ScrolledText
 except:
-	from tkinter.scrolledtext import ScrolledText
-import xerox
+    from tkinter.scrolledtext import ScrolledText
+
+curr_dir = os.getcwd()
+
+
+def clear_session(root):
```
Comment (nikhilkumarsingh) [inline clix/gui.py]: When is this function triggered?
Menubar is not showing up on gui window.

**Comment by nikhilkumarsingh** [comment]
```
Traceback (most recent call last):
  File "/usr/local/bin/clix", line 11, in <module>
    load_entry_point('clix==1.0.6', 'console_scripts', 'clix')()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 560, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2648, in load_entry_point
    return ep.load()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2302, in load
    return self.resolve()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2308, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python2.7/dist-packages/clix/__init__.py", line 1, in <module>
    from .clix import *
  File "/usr/local/lib/python2.7/dist-packages/clix/clix.py", line 32, in <module>
    with open(curr_dir + "/clix/config", "wb") as f:
IOError: [Errno 2] No such file or directory: '/home/nikhil/clix/config'
```

Please fix this error as well.

**Comment by akansh97531** [comment]
i resolved the file error and for menubar not showing up , probably uninstall and install will do it

**Comment by nikhilkumarsingh** [comment]
The error still persists because the error is caused in `except` block.
```
Traceback (most recent call last):
  File "/usr/local/bin/clix", line 11, in <module>
    load_entry_point('clix==1.0.6', 'console_scripts', 'clix')()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 560, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2648, in load_entry_point
    return ep.load()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2302, in load
    return self.resolve()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2308, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python2.7/dist-packages/clix/__init__.py", line 1, in <module>
    from .clix import *
  File "/usr/local/lib/python2.7/dist-packages/clix/clix.py", line 26, in <module>
    with open(curr_dir + "/clix/config", "wb") as f:
IOError: [Errno 2] No such file or directory: '/home/nikhil/clix/config'
```

**Comment by nikhilkumarsingh** [comment]
Try to test your package by running it from some other directories as well.

**Comment by akansh97531** [comment]
@nikhilkumarsingh  all error resolved

File: setup.py [hunk 2]
```diff
@@ -29,7 +29,7 @@ def readme():
 	  author = 'Nikhil Kumar Singh',
 	  author_email = 'nikhilksingh97@gmail.com',
 	  license = 'MIT',
-	  packages = ['clix'],
+	  packages = ['clix',''],
```
Comment (nikhilkumarsingh) [inline setup.py]: Why have you done this?
 @akansh97531

**Comment by nikhilkumarsingh** [comment]
@akansh97531 I am still getting the same error.
I advise you to open clix from a directory other than package directory. In that case, 
```
    with open(curr_dir + "/clix/config", "wb") as f:
IOError: [Errno 2] No such file or directory: '/home/nikhil/clix/config'
```
this error is obtained.

File: setup.py (hunk 2 above)
Comment (akansh97531) [inline setup.py]: this is for icon.png to be included in package directory

Comment (nikhilkumarsingh) [inline setup.py]: Well I don't know if this is the best way to do this. Can you send me the link where you found this solution?

**Comment by akansh97531** [comment]
this error should not be there now because i replaced above line with : 
with open(os.path.join(os.path.dirname(__file__),'config'), "rb") as f:
    key_binding = pickle.load(f)
which loads config from package directrory 
and i tested it opening from various dir. , it's all working fine

**Comment by nikhilkumarsingh** [comment]
Ya its working fine now.

**Comment by akansh97531** [comment]
@nikhilkumarsingh  i am now working on async. updation of gui.
the thing is i will have to run pyxhook and tkinter gui in seperate threads , since tkinter mainloop stops everything , and update gui accordingly

File: setup.py (hunk 2 above)
Comment (akansh97531) [inline setup.py]: I don't find it anywhere , i read the official documentation on packaging , and tried this.
I can also do it other way , including icon.png in clix folder and specifying it in MANIFEST.IN

Comment (nikhilkumarsingh) [inline setup.py]: Yes please do it in correct manner. Add it to MANIFEST.in

**Comment by nikhilkumarsingh** [comment]
Ok. First create a new issue for that,

**Comment by akansh97531** [comment]
@nikhilkumarsingh  please merge my last commit as well

**Pull request merged**